[2025-06-04T10:02:03.955+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: auto_pull_dags_from_github.git_pull_latest_dags scheduled__2025-06-04T09:00:00+00:00 [queued]>
[2025-06-04T10:02:03.962+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: auto_pull_dags_from_github.git_pull_latest_dags scheduled__2025-06-04T09:00:00+00:00 [queued]>
[2025-06-04T10:02:03.962+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-06-04T10:02:03.973+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): git_pull_latest_dags> on 2025-06-04 09:00:00+00:00
[2025-06-04T10:02:03.979+0000] {standard_task_runner.py:60} INFO - Started process 2363 to run task
[2025-06-04T10:02:03.983+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'auto_pull_dags_from_github', 'git_pull_latest_dags', 'scheduled__2025-06-04T09:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/auto_pull_dags.py', '--cfg-path', '/tmp/tmpplb93li0']
[2025-06-04T10:02:03.985+0000] {standard_task_runner.py:88} INFO - Job 48: Subtask git_pull_latest_dags
[2025-06-04T10:02:04.056+0000] {task_command.py:423} INFO - Running <TaskInstance: auto_pull_dags_from_github.git_pull_latest_dags scheduled__2025-06-04T09:00:00+00:00 [running]> on host 036d21481f2d
[2025-06-04T10:02:04.124+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='aswin.b@sganlytics.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='auto_pull_dags_from_github' AIRFLOW_CTX_TASK_ID='git_pull_latest_dags' AIRFLOW_CTX_EXECUTION_DATE='2025-06-04T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-04T09:00:00+00:00'
[2025-06-04T10:02:04.130+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-04T10:02:04.132+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dags && git pull origin main']
[2025-06-04T10:02:04.142+0000] {subprocess.py:86} INFO - Output:
[2025-06-04T10:02:04.144+0000] {subprocess.py:93} INFO - /usr/bin/bash: line 1: git: command not found
[2025-06-04T10:02:04.145+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2025-06-04T10:02:04.153+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2025-06-04T10:02:04.158+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=auto_pull_dags_from_github, task_id=git_pull_latest_dags, execution_date=20250604T090000, start_date=20250604T100203, end_date=20250604T100204
[2025-06-04T10:02:04.172+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/email.py:154: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2025-06-04T10:02:04.173+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-06-04T10:02:06.372+0000] {email.py:281} INFO - Sent an alert email to ['aswin.b@sganlytics.com']
[2025-06-04T10:02:08.580+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 48 for task git_pull_latest_dags (Bash command failed. The command returned a non-zero exit code 127.; 2363)
[2025-06-04T10:02:08.601+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-04T10:02:08.621+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
